react/src/components/atoms/forms/CompoundSlider/index.js

```diff
@@ -34,14 +40,15 @@ const Handle = (props) => {
       top: `${percent}%`
     };
   }
-  if (skipped) {
-    divProps.tabIndex = -1;
-  }
   return(
     <button className="ma__slider-handle" {...divProps}>
-      <div className="ma__slider-handle-value">
-        {roundedValue}
-      </div>
+      { props.displayValueFormat && (
+        <div className="ma__slider-handle-value">
+          { props.displayValueFormat === 'percentage' ? `${roundValue(step, value * 100, 0)}%` : roundedValue }
```

Comment:
Isn't value a string here? Might be easier to use numbro, as it handles percents and rounding. Example:
`numbro(value).format({
    output: "percent",
    mantissa: 0
});`
http://numbrojs.com/format.html#percentages